### PR TITLE
Update browser support page

### PR DESCRIPTION
**Pull request by sylwiavargas**
# PR Description

Important update about Safari 16.4 TP.

### Summary of my changes and explanation of my decisions

- Changed the page description
- Changed the Safari section
- Added mention of safari in "Web Platform requirements"

-----
<a href="https://stackblitz.com/~/github.com/stackblitz/docs/tree/web-publisher/sylwiavargas/browser-support">Review PR in StackBlitz Codeflow</a> _Submitted with [StackBlitz Web Publisher](https://developer.stackblitz.com/codeflow/integrating-web-publisher)._

### CODE REVIEW

File: docs/platform/webcontainers/browser-support.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: &title WebContainers Browser Support
-description: &description For WebContainers, we support desktop Chromium-based browsers out of the box, and Firefox in alpha state.
+description: &description For WebContainers, we support desktop Chromium-based browsers out of the box, Safari 16.4 TP, and Firefox in alpha state.
```
Comment (SamVerschueren) [inline docs/platform/webcontainers/browser-support.md:3]: I'm not that good at comma's, but do we need to remove this one to indicate that Safari is also alpha?

```suggestion
description: &description For WebContainers, we support desktop Chromium-based browsers out of the box, Safari 16.4 TP and Firefox in alpha state.
```

File: docs/platform/webcontainers/browser-support.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ head:
 
 _Last update: February 2023_
 
-**TL;DR** For WebContainers, we support desktop Chromium-based browsers out of the box, and Firefox in alpha state. If you have issues with supported browsers, [check your browser configuration](/platform/webcontainers/browser-config).
+**TL;DR** For WebContainers, we support desktop Chromium-based browsers out of the box, Safari 16.4 TP, and Firefox in alpha state. If you have issues with supported browsers, [check your browser configuration](/platform/webcontainers/browser-config).
```
Comment (SamVerschueren) [inline docs/platform/webcontainers/browser-support.md:15]: Same comment as above

**Review by SamVerschueren — CHANGES_REQUESTED** [review]
Thank you so much for updating these docs! 🙏 

Having some questions/comments. But like the update!

File: docs/platform/webcontainers/browser-support.md (hunk 1 above)
Comment (sylwiavargas) [reply docs/platform/webcontainers/browser-support.md:3]: Oh I see. 

```suggestion
description: &description For WebContainers, we support desktop Chromium-based browsers out of the box, as well as Safari 16.4 TP and Firefox, which both are in alpha state.
```

File: docs/platform/webcontainers/browser-support.md (hunk 2 above)
Comment (sylwiavargas) [reply docs/platform/webcontainers/browser-support.md:15]:
```suggestion
**TL;DR** For WebContainers, we support desktop Chromium-based browsers out of the box, as well as Safari 16.4 TP and Firefox, which both are in alpha state.. If you have issues with supported browsers, [check your browser configuration](/platform/webcontainers/browser-config).
```

File: docs/platform/webcontainers/browser-support.md (hunk 1 above)
Comment (SamVerschueren) [reply docs/platform/webcontainers/browser-support.md:3]: Yes, I like that!

File: docs/platform/webcontainers/browser-support.md [hunk 3]
```diff
@@ -62,12 +62,14 @@ In addition to this, there might be [other runtime incompatibilities](#runtime-d
 
 ## Safari
 
-Safari recently shipped support for `SharedArrayBuffer` and cross-origin isolation in a somewhat [buggy state](https://bugs.webkit.org/show_bug.cgi?id=238442). In addition to this, it is still lacking a few other features which prevents us from shipping a working environment such as:
+Starting with Safari 16.4 TP, WebContainers fully work in Safari.
```
[the remark this answers is not in the document]
Comment (sylwiavargas) [reply docs/platform/webcontainers/browser-support.md:65]:
```suggestion
Starting with Safari 16.4 TP,  WebContainers are in beta for Safari.
```

[the remark this answers is not in the document]
Comment (sylwiavargas) [reply docs/platform/webcontainers/browser-support.md:65]: Thank you!

File: docs/platform/webcontainers/browser-support.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: &title WebContainers Browser Support
-description: &description For WebContainers, we support desktop Chromium-based browsers out of the box, and Firefox in alpha state.
+description: &description For WebContainers, we support desktop Chromium-based browsers out of the box, as well as Safari 16.4 TP and Firefox, which both are in alpha state.
```
Comment (sylwiavargas) [inline docs/platform/webcontainers/browser-support.md:3]:
```suggestion
description: &description For WebContainers, we support desktop Chromium-based browsers out of the box, as well as Safari 16.4 TP and Firefox, which both are in beta.
```

Comment (sylwiavargas) [reply docs/platform/webcontainers/browser-support.md:3]: @SamVerschueren ?

File: docs/platform/webcontainers/browser-support.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ head:
 
 _Last update: February 2023_
 
-**TL;DR** For WebContainers, we support desktop Chromium-based browsers out of the box, and Firefox in alpha state. If you have issues with supported browsers, [check your browser configuration](/platform/webcontainers/browser-config).
+**TL;DR** For WebContainers, we support desktop Chromium-based browsers out of the box, as well as Safari 16.4 TP and Firefox, which both are in alpha state. If you have issues with supported browsers, [check your browser configuration](/platform/webcontainers/browser-config).
```
Comment (sylwiavargas) [inline docs/platform/webcontainers/browser-support.md:15]:
```suggestion
**TL;DR** For WebContainers, we support desktop Chromium-based browsers out of the box, as well as Safari 16.4 TP and Firefox, which both are in beta. If you have issues with supported browsers, [check your browser configuration](/platform/webcontainers/browser-config).
```

Comment (SamVerschueren) [reply docs/platform/webcontainers/browser-support.md:15]: Thanks for making these consistent! 👍

File: docs/platform/webcontainers/browser-support.md [hunk 6]
```diff
@@ -62,12 +62,14 @@ In addition to this, there might be [other runtime incompatibilities](#runtime-d
 
 ## Safari
 
-Safari recently shipped support for `SharedArrayBuffer` and cross-origin isolation in a somewhat [buggy state](https://bugs.webkit.org/show_bug.cgi?id=238442). In addition to this, it is still lacking a few other features which prevents us from shipping a working environment such as:
+Starting with Safari 16.4 TP,  WebContainers are in beta for Safari.
```
Comment (SamVerschueren) [inline docs/platform/webcontainers/browser-support.md:65]:
```suggestion
Starting with Safari 16.4 TP, WebContainers are in beta for Safari.
```

There's an additional space 🙈

**Review by SamVerschueren — CHANGES_REQUESTED** [review]
One small thing

File: docs/platform/webcontainers/browser-support.md (hunk 6 above)
Comment (sylwiavargas) [reply docs/platform/webcontainers/browser-support.md:65]: thank you, detective! 🕵️‍♂️

**Review by SamVerschueren — APPROVED** [review]
